src/timer.c

```diff
@@ -119,7 +119,10 @@ void TIMER_Power(){
 
     elevator = 2 == mode ? abs(CHAN_ReadInput(INP_THROTTLE)) : abs(CHAN_ReadInput(INP_ELEVATOR));
     new_throttle = 2 == mode ?  abs(CHAN_ReadInput(INP_ELEVATOR)) : abs(CHAN_ReadInput(INP_THROTTLE));
-    new_throttle = abs(new_throttle - throttle);
+    if (new_throttle > throttle)
```

Comment:
this is warning that both are unsigned and result is unsigned as well. so we have to explict check to avoid use abs, which doesn't work.

Reply:
yes, this one looks good